libcodechecker/libhandlers/check.py

```diff
@@ -94,13 +96,32 @@ def add_arguments_to_parser(parser):
     Add the subcommand's arguments to the given argparse.ArgumentParser.
     """
 
+    def is_ctu_capable():
```

Comment:
Shouldn't we move this to a separate module, this is used by `check` and `analyze` commands if I'm correct right?